frontend/src/pages/projectPage/TaskItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import "./TaskItem.css";
+import Calendar from "../../assets/icons/Calendar.png";
+import Assigned from "../../assets/icons/assigned task.png";
+import Circle from "../../assets/icons/Danger Circle.png";
+
+interface Project {
+  id: number;
+  title: string;
+  thumbnail_link: string;
+  date_of_creation: string;
+  amount_of_tasks: number;
+}
```

Comment:
duplicate interface also found in TaskList. Did you sync with develop? I think someone already made a Project class